web-portal/backend/src/utils/utils.service.ts

```diff
@@ -228,4 +230,11 @@ export class UtilsService {
     }));
   }
 
+  @Cron(CronExpression.EVERY_HOUR)
+  async reconciliation(){
+    console.log("Started Reconciliation at " + new Date().toLocaleString());
+    () => this.watchEvent(25000);
+    console.log("Finished Reconciliation at " + new Date().toLocaleString());
+  }
+
```

Comment:
I don't quite understand this, is this just backup in case things are missed in the other check?

Reply:
Yes